### PR TITLE
Feat/header component

### PR DESCRIPTION
**Pull request by kputnins**

### CODE REVIEW

File: src/components/Header/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  logoUrl?: string;
+  children?: ReactNode;
+}
+
+export const Header: FC<HeaderProps> = ({
+  logoUrl,
+  children,
+  ...otherProps
+}) => (
+  <header className={`${prefix}-header`} {...otherProps}>
```
Comment (lienputnina) [inline src/components/Header/Header.tsx:22]: {...otherProps} are meant HTMLProps?

File: src/components/Header/Header.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    <a id="logo" href={logoUrl || '/'}>
+      <img src={logo} alt="Scrabble logo" />
```
Comment (lienputnina) [inline src/components/Header/Header.tsx]: Will it be ok to have a fixed logo img for a reusable component library?

**Review by lienputnina — CHANGES_REQUESTED** [review]

File: src/components/Header/Header.tsx (hunk 1 above)
Comment (kputnins) [reply src/components/Header/Header.tsx:22]: Basically yest.
To be more precise all other props not destructed from the `props` parameter (provided to the component) that are of the expected type of `HTMLProps<HTMLDivElement>`

File: src/components/Header/Header.tsx (hunk 2 above)
Comment (kputnins) [reply src/components/Header/Header.tsx]: Very good point - this should be made as the default logo but a way to provide a custom one should be provided

Comment (kputnins) [reply src/components/Header/Header.tsx]: @lienputnina Made updates - added prop to provide custom logo. Added story and unit tests

**Review by lienputnina — APPROVED** [review]

**Pull request merged**